quantumdb-postgresql/src/main/java/io/quantumdb/core/planner/SelectiveMigratorFunction.java

```diff
@@ -145,7 +145,7 @@ private static MigratorFunction createUpdateMigrator(RefLog refLog, Table source
 		createStatement.append("		  WHERE  " + primaryKeyCondition + ";");
 		createStatement.append("	  EXCEPTION WHEN unique_violation THEN END;");
 		createStatement.append("	END LOOP;");
-		createStatement.append("  RETURN CONCAT('(', " + primaryKeyColumnNames.stream().map(value -> "r." + quoted(value)).collect(Collectors.joining(", ")) + ", ')');");
+		createStatement.append("  RETURN CONCAT('(', " + primaryKeyColumnNames.stream().map(value -> "r." + quoted(value)).collect(Collectors.joining(",',', ")) + ", ')');");
```

Comment:
An example I ran with multiple primary keys got an index out of bounds exception, it turned out this function returned (101) instead of (10,1). It now adds comma's between these values